### DISCUSSION TITLE
这项目开发者存在严重的偏见

### DISCUSSION THREAD

**Discussion by ungluee** [c0re100/qBittorrent-Enhanced-Edition#593] [General]
1、国内bt客户端一刀切，你能证明国内软件全都会吸血？做开发的一点也不严谨。
2、反吸血功能很烂，我用bitcomet连接了一个qBittorrent Enhanced客户端，完成度99.2%，还连接了另一个bitcomet的长效种子完成度100%，长效种子网速很慢，按理来说，文件本来就冷门，我只能从qBittorrent Enhanced快速下载，并不能算吸血行为，并且我下载完他的99.2%，我可以下载剩下的0.8%传给他，但是这傻逼qBittorrent Enhanced客户端我下到32%的时候不传给我了，一直0k，我只能以40k的速度从另一个人那里下载，最后实在下载不动，我给他删了，99.2%的那个人下了一天，到我删除任务的时候还是99.2%，何必呢？我真服了。

**Comment by SeaHOH** [comment]
> 这傻逼qBittorrent Enhanced客户端我下到32%的时候不传给我了

有没有可能别人不止有你这一个下载任务，带宽都给你了其它种子活该没上传？

**Reply by ungluee** [reply]
那也不可能1k都不给吧，这明显就不正常。

**Reply by SeaHOH** [reply]
老种还讲究啥，不爽不要玩。看你这描述最多不超过一个月，快可能不到一周就能下完，别人都传你 32% 了，还跑啥？如果真判吸血，还能让你有机会吐嘈别人下一天的 99.2%，根本就连不上好吧。电驴玩过没，几个月是常态，一年也不稀奇。完成时长方面，BT 老种和这区别也不大，绝大部分 (死种) 还不如电驴呢。

**Reply by catvox** [reply]
用代理吧或者ipv6，反正我基本都下载完的。

**Comment by qwqcwcawa** [comment]
1、国内bt客户端只封了迅雷 QQ旋风 影视先锋还有peer id 不符合bt规范的客户端
2、你说的问题不是反吸血造成的，bt中这种情况很常见的

**Comment by ungluee** [comment]
希望屏蔽的迅雷客户端可以把XunLei0100排除掉，我是真的看到过XunLei0100做种，而且上传速度还不低。

**Reply by SeaHOH** [reply]
你直接用渣雷不行吗，为什么要为难自己？

**Reply by Mightnire** [reply]
https://github.com/PBH-BTN/PeerBanHelper/issues/254#issuecomment-2439855839 到底是什么改变了你？

**Comment by 19890604-boop** [comment]
没把中国的IP地址都屏蔽已经算这位开发者有超出常人的博爱了。知足吧。

**Comment by Zerorigin** [comment]
![qbee](https://github.com/user-attachments/assets/ffbb247d-5b92-4b8b-ba6a-a6cd6f58dfc2)

挂了快两个月没下完，我都不敢说话。

就像我之前在其它 issue 说的，BT 客户端的网络连接和上传带宽是轮流分配给排队的客户端的，
所以自己的客户端没从别人那边得到上传流量是很正常的现象，所以别那么心急。
要不就只能用迅雷从迅雷社区吸完血，再补挂回正常的 BT 社区回血。

另外最近一年左右，整个 BT 网络（DHT + PeX）被搞 PCDN 刷流的到处吸血，
造成死种速度明显比以前快，就连热门种子的做种人数也明显比以前少了，
整个 BT 网络现在只能靠做种者开 [PeerBanHelper](https://github.com/PBH-BTN/PeerBanHelper) 辅助，暂且苟活。

最后补充些知识：
能不能获得别人的上传速度，除了上面说的，另一些比较重要的是，
首先要优先确保自己本地的 IPv4 和 IPv6 网络都是正常的，包括不限于防火墙设置、PPPoE 拨号设置、IPv6 地址更新通告机制设置等
其次要确保自己的客户端各项网络连接参数是否正确配置。

上面（这个是单栈 v6）这个种子，还是在我下不动的情况下，另外开了一台双栈公网（v4+v6）的机子挂着一起下，
然后双栈公网这台下到（从社区内一个单栈 v4 做种者那里慢慢下了好几天的，也就 100KB/s 左右的速度）后，
把数据传给单栈的这台的（单栈这台没开 [PeerBanHelper](https://github.com/PBH-BTN/PeerBanHelper) 前，被各种吸血，但就是下不动）。

**Comment by Voodookid81** [comment]
> 1、国内bt客户端一刀切，你能证明国内软件全都会吸血？做开发的一点也不严谨。 2、反吸血功能很烂，我用bitcomet连接了一个qBittorrent Enhanced客户端，完成度99.2%，还连接了另一个bitcomet的长效种子完成度100%，长效种子网速很慢，按理来说，文件本来就冷门，我只能从qBittorrent Enhanced快速下载，并不能算吸血行为，并且我下载完他的99.2%，我可以下载剩下的0.8%传给他，但是这傻逼qBittorrent Enhanced客户端我下到32%的时候不传给我了，一直0k，我只能以40k的速度从另一个人那里下载，最后实在下载不动，我给他删了，99.2%的那个人下了一天，到我删除任务的时候还是99.2%，何必呢？我真服了。

**迅雷0.01.8 无论下载什么，进度永远是0%！ 用这个吸血的人还多得很！**
